packages/types/CHANGELOG.md

```diff
@@ -9,7 +9,9 @@ All logs must start with the format: [x.y.z] - yyyy-mm-dd
 
 ## [Unreleased]
 
+## [1.2.0] - 2022-07-27
 ### Changed
+- Update `SubstrateBlockFilter` with new `modulo` filter (#1196)
 - Make `SubstrateExtrinsic` and `SubstrateEvent` types generic. This allows specifying the data/args type rather than being provided with `Codec[]` or `AnyTuple` (#1194)
```

Comment:
```suggestion
- Make `SubstrateExtrinsic` and `SubstrateEvent` types generic. This allows specifying the data/args type rather than being provided with `Codec[]` or `AnyTuple`. (#1194)
```